WikipediaUnitTests/Code/UIView+VisualTestSizingUtils.m

```diff
@@ -17,6 +17,7 @@ - (CGRect)wmf_sizeThatFitsScreenWidth {
 }
 
 - (CGRect)wmf_sizeThatFitsScreenWidthForOrientation:(UIInterfaceOrientation)orientation {
+    WMF_TECH_DEBT_TODO(use nativeBounds of mainScreen)
```

Comment:
because some of the tests are meant to happen across multiple lines, which isn't true anymore if the view is wider. it's just more work that's not worth doing atm